save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/SelectForm.kt

```diff
@@ -104,7 +104,11 @@ fun selectFormRequired(
                 }
             }
 
-            if (props.validInput == false) {
+            if (elements.isEmpty()) {
+                div("invalid-feedback d-block") {
+                    +"You don't have organizations"
```

Comment:
Maybe `You don't have access to any organizations`?

Reply:
Fixed